consensus/config/src/committee.rs

```diff
@@ -59,17 +59,27 @@ impl Committee {
     pub fn new_for_test(
```

Comment:
Maybe we can just update this function to accept stakes instead of `num_of_authorities`.